### DISCUSSION TITLE
Testimonials

### DISCUSSION THREAD

**Discussion by Mimetis** [Mimetis/Dotmim.Sync#516] [General]
I was wondering... 

Would you be interested in sharing what you've done with DMS and your application using it, in a special dedicated page in the official DMS documentation https://dotmimsync.readthedocs.io/ ?

I mean, I'm considering to add a new section to the doc (just after [Overview ](https://dotmimsync.readthedocs.io/Overview.html) and before [How does it work](https://dotmimsync.readthedocs.io/HowDoesItWorks.html)) called something like **They are using it** which will contains testimonials from people using `DMS` in their application / company.

The only thing I need from **you** is a markdown file describing your project, with a feedback on DMS. Maybe one or 2 images, code, links; whatever :)

It could be a simple paragraph, or a full page, you can choose the format and size your prefer.

Interested ?

If yes, feel free to create your own testimonials here, that we can review together, or send it directly to me.

@Mimetis @VladislavAntonyuk @vunder @VagueGit, @maxrodro, @sylverbee, @gentledepp, @workgroupengineering; @gb0o

**Comment by sylverbee** [comment]
@Mimetis I'll send you my testimonial by mail

**Comment by tommy-uytterhaegen** [comment]
@Mimetis I have sent you my testimonial by mail as well. Was it what you have in mind?

**Reply by Mimetis** [reply]
Yes, I received it, thanks a ton !
Let me the day to see it and get back to you :)
